### PR TITLE
Use filter_horizontal for many-to-many fields

### PR DESCRIPTION
**Pull request by adam-thomas**
This makes them a lot easier to use, especially for large lists.

@incuna/backend quick review please?

### CODE REVIEW

File: groups/admin.py [hunk 1]
```diff
@@ -3,5 +3,19 @@
...
+
+class DiscussionAdmin(admin.ModelAdmin):
+    filter_horizontal = ('subscribers', 'ignorers')
+
+    class Meta:
+        model = Discussion
+
+
+admin.site.register(Group, GroupAdmin)
+admin.site.register(Discussion, DiscussionAdmin)
```
Comment (meshy) [inline groups/admin.py]: As we're using django `1 .8`, these can be replaced with decorators!
```python
@admin.register(Discussion)
class DiscussionAdmin(...):
    ...
```

**Comment by meshy** [comment]
Looks good to go to me (barring my comment above).

File: groups/admin.py (hunk 1 above)
Comment (adam-thomas) [inline groups/admin.py]: sweet

**Comment by adam-thomas** [comment]
Thanks :) updated!

**Comment by adam-thomas** [comment]
Cheers :)

**Pull request merged**